tests/test_lm/test_unit/test_jsrun.py

```diff
@@ -46,18 +46,20 @@ def test_create_resource_set_file(mocked_init, mocked_method, mocked_raise_on):
 
 # ------------------------------------------------------------------------------
 #
+def _create_resource_set_file(self, slots, uid, sandbox):
+    return 'rs_layout_cu_%06d' % (int(uid.split('_')[1]))
 @mock.patch.object(JSRUN, '__init__', return_value=None)
 @mock.patch.object(JSRUN, '_configure', return_value='jsrun')
 @mock.patch.object(JSRUN, '_create_resource_set_file',
-                          return_value='rs_layout_cu_000000')
+                           _create_resource_set_file)
```

Comment:
Is this a repetition?

Reply:
No - basically it mocks the member method `_create_resource_set_file` with the local method of the same name defined just above the call (the previous mock was wrong).  Thanks for checking! :-)